code/modules/antagonists/disease/disease_abilities.dm

```diff
@@ -5,6 +5,9 @@ is currently following.
 */
 
 GLOBAL_LIST_INIT(disease_ability_singletons, list(
```

Comment:
honestly this should probably just be changed to a typecache.

Reply:
me can't read code, are typecaches the same as new type/1, new type/2, etc.

Reply:
There is no reason for this to be a typecache.